aggregators/aggregator.go

```diff
@@ -197,6 +197,21 @@ func (a *Aggregator) AggregateCombinedMetrics(
 	default:
 	}
 
+	if cmk.ProcessingTime.Before(a.processingTime.Add(-a.cfg.Lookback)) {
```

Comment:
@marclop Not sure if this should be reported here. I think the client can retry this on failure and can cause the metrics to be skewed. Maybe just error logging is enough here?

Reply:
Would it make sense to revert the responsibility and have the client report the failure instead?

Reply:
For posterity, after discussing with @marclop we decided that silently dropping the errors for very old events with warning level logs and failure metrics is better.